flows/deployment.py

```diff
@@ -0,0 +1,25 @@
+from prefect import flow, get_run_logger
+from prefect.deployments import Deployment
+from prefect.utilities.callables import parameter_schema
+
+
+@flow
+def hello(name: str = "world"):
+    get_run_logger().info(f"Hello {name}!")
+
+
+if __name__ == "__main__":
+    # Create deployment
+    if hasattr(Deployment, "build_from_flow"):
```

Comment:
I think this should bifurcate on Prefect version number rather than presence of the attribute to avoid hidden breaking changes.